src/App.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import Login from './components/auth/login';
+import AddHotel from './components/hotels/AddHotel';
+import { loginActions } from './redux/login/loginReducer';
+import LogedUsers from './components/accessibility/LogedUsers';
+import DetailsForm from './components/Details/detailsForm';
+import IsAdmin from './components/accessibility/isAdmin';
+import Reservations from './components/reservations/Reservations';
+import RemoveHotel from './components/hotels/removeHotel/RemoveHotel';
+import SignUp from './components/auth/register';
+import Home from './components/Home';
+import Index from './components/Main';
+import Reserve from './components/reservations/Reserve';
+
+const App = () => {
```

Comment:
- [ ] When I create a new account, it seems I already have lots of reservations taking place:


![image](https://user-images.githubusercontent.com/57421823/213681178-e08e6fd1-4082-4cdf-a93b-200b25376a4d.png)

If I create a reservation, it adds to the collection that is already there.

If I create another account with a different user, the same happens.  It seems that all the users share the same reservations and this is not the intended functionality.  

If this is a bug, kindly fix it so the project is ready for the final evaluation. 💪

Reply:
@Meltrust Hi! ✋🏼 Thank you for your feedback, we solved this issue already.